### ISSUE TITLE
PSF fitting in crowded fields should not be able to fit source locations independently.

### ISSUE THREAD

**Issue by christinahedges** [KeplerGO/lightkurve#135]
I don't quite agree with the way we do PSF photometry currently for crowded fields. If we make the assumption that targets are static and always maintain the same separation on the sky then we should be fitting for one spacecraft motion vector as a function of time and one 'scene' location variable as a function of x and y.

(This assumption is not strictly true, but a good approximation for small target pixel files.)

This would mean that in we would take in a prior on RA/Dec for each source and a prior on motion and fit those.

This is essentially what's already being done in #95 and #111, we just need some little tweaks to make sure that in crowded fields the RA/Dec of the scene is fit, not individual sources.

In other words this should not be possible

![bad](https://user-images.githubusercontent.com/14965634/41058220-124cc78a-697e-11e8-9c5f-e67998b8022d.gif)

Only this should be possible

![good](https://user-images.githubusercontent.com/14965634/41058231-193c3274-697e-11e8-9702-8245e745a46d.gif)

**Comment by christinahedges** [comment]
@gully this is what I mean. @mirca do you have thoughts on this?

**Comment by mirca** [comment]
I couldn't agree more with this!

If you don't want to touch the RA/Dec part initially, I think the way to implement this would be:

1) use some good frame to jointly fit the centroids of all sources as ``(x1, y1), (x1 + dx2, y1 + dy2), (x1 + dx3, y1 + dy3)...``
2) for the rest of the frames, fix ``dx2, dy2, dx3, dy3...`` and only fit for ``(x1, y1)``.

This reduces the number of parameters fitted per frame by ``2(N - 1)`` where ``N`` is the number of sources.

**Comment by mirca** [comment]
I believe this has been accounted for in @barentsen's refactoring. I'm closing it, feel free to re-open tho.
